static/js/biomedical/protein/protein_protein_interaction_graph.tsx

```diff
@@ -111,14 +125,47 @@ export class ProteinProteinInteractionGraph extends React.Component<
     if (this.state.graphData === null) {
       return null;
     }
+    const data = this.state.graphData.linkDataNested
+      .slice(0, this.state.depth + 1)
+      .flat(1);
     return (
-      <>
-        <div id={GRAPH_ID} />
-        <ProteinProteinInteractionTable
-          data={this.state.graphData.linkDataNested
-            .slice(0, this.state.depth + 1)
-            .flat(1)}
-        />
+      <div className="ppi-container">
+        <Row className="justify-content-end">
+          <ButtonGroup>
+            <Button
+              className="btn btn-sm btn-light shadow-none"
+              onClick={() =>
+                this.setState({ showTableView: !this.state.showTableView })
+              }
+            >
+              <i className="material-icons align-middle">
+                {this.state.showTableView ? "hub" : "table_restaurant"}
```

Comment:
maybe we can use this icon instead for table: https://fonts.google.com/icons?selected=Material+Symbols+Outlined:table_chart

Reply:
great selection! yes will do!